shm/Game.hpp

```diff
@@ -0,0 +1,54 @@
+#pragma once
+
+#include "DefaultCargo.hpp"
+#include "Map.hpp"
+#include "Player.hpp"
+#include "Store.hpp"
+#include "Fruit.hpp"
+#include "DryFruit.hpp"
+
+#include <array>
+#include <cmath>
+#include <iostream>
+
+constexpr size_t start_money = 1'000;
+constexpr size_t game_days = 100;
+constexpr size_t final_goal = 2'000;
+
+// Return by value to give up ownership
+std::unique_ptr<Cargo> generateCargo();
+
+#include "Command.hpp"
+
+class Game {
+public:
+    Game(size_t startMoney, size_t daysLimit, size_t goal)
+        : startMoney_(startMoney), daysLimit_(daysLimit), goal_(goal) {}
+
+    void startGame();
+    void init();
+
+    constexpr static size_t startingPlayerCargoNumber = 5;
+    constexpr static size_t storeCargoNumber = 5;
```

Comment:
should be at the top also make it inline, because in some compiler you get error when you define variable inside class